cactusbot/services/beam/handler.py

```diff
@@ -79,17 +77,23 @@ def __init__(self, channel):
     async def handle_liveloading(self, packet):
         """Handle liveloading packets."""
 
+        packet = json.loads(packet)
+
         data = packet.get("data")
-        if not isinstance(data, list) or not isinstance(data[0], str):
+        if not isinstance(data, dict):
+            return
+
+        event = data["channel"].split(":")[-1:][0]
+        data = data.get("payload")
+        if not isinstance(data, dict):
             return
 
-        event = re.match(self.liveloading.INTERFACE_EXPR, data[0])
         if event is None:
             return
 
-        event = event.groups()
-        if event[1] in self.liveloading_events.get(event[0], {}):
-            await self.liveloading_events[event[0]][event[1]](data[1])
+        if data.get("user"):
```

Comment:
Better practice would be to use `if "user" in data`, for the following reason:

```python
if data.get("user"):
    print("Found `user`.")
```
`data = {"user": "Innectic"}` - prints.
`data = {"user": None}` - doesn't print.
`data = {"user": False}` - doesn't print.

`"user" in data` just checks for the `"user"` key in `data` - in addition to being more accurate, it uses less computing power. :smile: